### ISSUE TITLE
Support Kubernetes `v1.25` Seeds

### ISSUE THREAD

**Issue by shafeeqes** [gardener/gardener#6893]
**How to categorize this issue?**

/area open-source
/kind enhancement
/topology seed

**What would you like to be added**:

Support for Kubernetes v1.25 Shoots were added in #6567.

Required tasks:
- [x] https://github.com/gardener/etcd-druid/issues/429
  - [x] Upgrade etcd-druid version in gardener/gardener > #7034 
- [x] gardener/hvpa-controller/issues/104
- [x] https://github.com/gardener/gardener/pull/7803
- [x] https://github.com/gardener/gardener/pull/7831

This issue is for tracking the same.

**Comment by rfranzke** [comment]
We should also tackle https://github.com/gardener/gardener/blob/017023657af9fc09853636775c15fe329fd3d4c8/test/e2e/managedseed/create_rotate_delete.go#L58-L59 once 1.25 seeds are supported.

**Comment by voelzmo** [comment]
I closed https://github.com/gardener/hvpa-controller/issues/104 with 'won't fix', our current plan is to have HVPA removed once we'll get to updating our Seeds to 1.25.

**Comment by shafeeqes** [comment]
> I closed [gardener/hvpa-controller#104](https://github.com/gardener/hvpa-controller/issues/104) with 'won't fix', our current plan is to have HVPA removed once we'll get to updating our Seeds to 1.25.

Is there an open issue for HVPA removal that we can link here?

**Comment by voelzmo** [comment]
It's https://github.com/gardener/backlog/issues/25 – but that probably only works for you and me, as it is in a private backlog.

**Comment by aminmr** [comment]
Hi friends,
Are there any updates on when we can use 1.25 for the seeds cluster?

**Comment by shafeeqes** [comment]
> Hi friends, Are there any updates on when we can use 1.25 for the seeds cluster?

We are still waiting for the HVPA controller to be dropped. It might take some time.
I believe if you turn off the HVPA feature gate in gardenlet, all the other components should work fine with v1.25. At least in my local (kind-based) setup, it does.

**Comment by voelzmo** [comment]
Once #7803 is merged, we should be able to officially enable k8s 1.25 support.
